src/web/src/current.ecpp

```diff
@@ -335,51 +339,45 @@ std::vector<uint32_t> asset_ids;
 
             if( one_measurement.first == "status.ups" )
             {
-                json += utils::json::jsonify (one_measurement.first, shared::upsstatus_to_string(one_measurement.second));
+              siJson.addMember(one_measurement.first) <<= shared::upsstatus_to_string(one_measurement.second);
             }
             else if (one_measurement.first == "operatingStatus" )
             {
-                json += utils::json::jsonify (one_measurement.first, s_os2string(one_measurement.second));
+                siJson.addMember(one_measurement.first) <<= s_os2string(one_measurement.second);
             }
             else
             {
-                json += utils::json::jsonify (one_measurement.first, one_measurement.second);
+                siJson.addMember(one_measurement.first) <<= one_measurement.second;
             }
-            json += ",";
         }
 
         // BIOS-951 -- begin
         if (persist::is_epdu (asset.item.subtype_id)) {
-            json += "\n    \"outlets\" : {";
+            cxxtools::SerializationInfo& sioutlets = siJson.addMember("outlets");
             for (const auto &it : outlet_properties) {
-                json += "\n        \"" + it.first + "\" : " + it.second.toJson () + ",";
+                cxxtools::SerializationInfo& siOutletProperties = sioutlets.addMember(it.first);
+
+                siOutletProperties.addMember("realpower") <<= it.second.realpower;
+                siOutletProperties.addMember("current") <<= it.second.current;
+                siOutletProperties.addMember("voltage") <<= it.second.voltage;
+
+                if(it.second.status.empty())
+                {
+                  siOutletProperties.addMember("status").setNull();
+                }
+                else
+                {
+                  siOutletProperties.addMember("status") <<= it.second.status;
+                }
             }
-
-            if (json.back () == ',') {
-                json.back () = '\n';
-                json += "    }";
-            }
-            else
-                json += '}';    // outlet_properties is empty
-
-            json += "\n,";
         }
         // BIOS-951 -- end
-
-        // replace last "," by "}"
-        json.back() = '}';
-        big_json += json;
-        big_json += ",";
     }
+    std::ostringstream outJsonStream(std::ios_base::out);
+    cxxtools::JsonSerializer serializer(outJsonStream);
+    serializer.serialize(si).finish();
+    big_json = outJsonStream.str().c_str();
 
-
-    // remove last ","
-    if (!big_json.empty())
-        big_json.pop_back();
 </%cpp>
-{
-   "current":[
```

Comment:
I think that if we decide to remove this, we have to take care that UI is updated for compatibility.